### PR TITLE
Add RemoteProductDir input param to ei and esb scripts

### PR DESCRIPTION
**Pull request by malshaniS**

### CODE REVIEW

File: team-ei/dev-wum-sce-test-wso2ei-6.2.0-full.yaml [hunk 1]
```diff
@@ -46,4 +46,5 @@ scenarioConfigs:
   description: "ei-scenarios"
   file: product-scenarios/test.sh
   inputParameters:
-    ProductVersion: "EI-6.2.0"
\ No newline at end of file
+    ProductVersion: "EI-6.2.0"
+    RemoteProductDir: "/usr/lib/wso2/wso2ei/6.2.0/wso2ei-6.2.0"
```
Comment (dulanjalidilmi) [inline team-ei/dev-wum-sce-test-wso2ei-6.2.0-full.yaml]: Shall we add a new line?

File: team-ei/dev-wum-sce-test-wso2ei-6.3.0-full.yaml [hunk 2]
```diff
@@ -46,4 +46,5 @@ scenarioConfigs:
   description: "ei-scenarios"
   file: product-scenarios/test.sh
   inputParameters:
-    ProductVersion: "EI-6.3.0"
\ No newline at end of file
+    ProductVersion: "EI-6.3.0"
+    RemoteProductDir: "/usr/lib/wso2/wso2ei/6.3.0/wso2ei-6.3.0"
```
Comment (dulanjalidilmi) [inline team-ei/dev-wum-sce-test-wso2ei-6.3.0-full.yaml]: Shall we add a new line?

**Review by dulanjalidilmi — CHANGES_REQUESTED** [review]

File: team-ei/dev-wum-sce-test-wso2ei-6.2.0-full.yaml (hunk 1 above)
Comment (malshaniS) [reply team-ei/dev-wum-sce-test-wso2ei-6.2.0-full.yaml]: Fixed. Please refer the latest commit.

File: team-ei/dev-wum-sce-test-wso2ei-6.3.0-full.yaml (hunk 2 above)
Comment (malshaniS) [reply team-ei/dev-wum-sce-test-wso2ei-6.3.0-full.yaml]: Fixed. Please refer the latest commit.

**Review by dulanjalidilmi — APPROVED** [review]

**Pull request merged**